xpk.py

```diff
@@ -2215,6 +2286,16 @@ def directory_path_type(value):
     ),
 )
 
+workload_create_parser_optional_arguments.add_argument(
```

Comment:
if you use `action='store_false'` then it will be easier to verify that the user sets this argument correctly. The user would then only need to say `--deploy-stacktrack-sidecar` instead of setting a string which they could accidently type wrong.

https://docs.python.org/3/library/argparse.html#action

Reply:
That's nice, I will make this change.